chatkit-core/src/main/kotlin/com/pusher/chatkit/messages/multipart/Message.kt

```diff
@@ -80,13 +83,13 @@ data class Attachment(
 )
 
 class UrlRefresher(
-        private val client: Instance
+        private val client: PlatformClient
```

Comment:
Why is there a need for this change?

Reply:
It's a change from the groundwork laid in a previous PR (which is as yet unreleased). We don't use the `Instance` level interface in the SDK, and I'd prefer to stick to the one layer of abstraction.

The only reason I initially chose `Instance` was because I was holding an absolute URL, not a relative path to the service, but I later found out I could still query an absolute URL using the `PlatformClient` interface.